pkg/repo/migrations/v3_4.go

```diff
@@ -15,45 +19,84 @@ import (
 // - if an installationID is present in the config its value is persisted to system_metadata.yaml.
 // - removes update.json if the file is present.
 // - removes repo.version.
-var V3Migration = repo.NewMigration(
+// - creates a new directory .bacalhau/compute_store/executions which contains the content of ./bacalhau/execution_store
+// - removes ./bacalhau/execution_store
```

Comment:
please stick to the exact naming and directory structure as in https://www.notion.so/expanso/Rethinking-Configuration-435fbe87419148b4bbc5119d413786eb?pvs=4#4671ae7e54704844a2b3058da6f6e1ed

Reply:
fixed in latest changes